ui/pages/home/home.component.js

```diff
@@ -640,7 +640,7 @@ export default class Home extends PureComponent {
             ? this.renderPopover()
             : null}
           <div className="home__main-view">
-            <MenuBar />
+            {process.env.MULTICHAIN ? null : <MenuBar />}
```

Comment:
Seems like there is only one instance of <MenuBar/> being used in the codebase. Does `MenuBar` only contain multichain related UI? I see a conditional inside of `MenuBar` that renders something If MULTICHAIN exists? If it's false it will never render? Should it just always be renderd?

Reply:
That's an amazing suggestion and question. Initially, when we were creating components for `Header` we added `GlobalMenu` in `MenuBar` which was supposed to be rendered only in Multichain UI. But the idea was to use the GlobalMenu component directly in the new Header component and remove it from MenuBar. Since we are not showing `MenuBar` in the Multichain UI. Hence it is conditionally rendered here.

Thanks for this suggestion, I have removed GlobalMenu from MenuBar.